wordpress-mobile/WordPress-Android
# Annual stats block in Stats

Fixes #9461 - the fluxC endpoint - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1209

This PR adds a block containing the "This Year" data. 
The "Average" date is rounded to one decimal point if it's < 9,999. If it's bigger, we use the standard shortened 10.5k format. 

The "View All" screen will be done in a separate PR.

To test:
* Go to Stats/Insights
* Scroll to the "This Year" block
* See that it's there and the data matches Calypso

![Screenshot_1555407620](https://user-images.githubusercontent.com/1079756/56204509-f1f7bc80-6047-11e9-875d-999f68734606.png)

Update release notes:
